.github/workflows/test.yml

```diff
@@ -1,54 +0,0 @@
-name: Test
```

Comment:
Can you keep this file but only have it do `test` for `macos-latest` and `windows-latest`? (ie remove lint, test-packages)

Reply:
Done